src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Helpers/DelegationHelper.cs

```diff
@@ -75,6 +75,54 @@ public static bool TryGetCoveredByUserIdFromMatch(List<AttributeMatch> match, ou
             return false;
         }
 
+        /// <summary>
+        /// Gets a int representation of the CoveredByUserId and CoverdByPartyId
+        /// </summary>
+        /// <returns>The CoveredByUserId and CoveredBy PartyId value</returns>
+        public static string GetCoveredByFromMatch(List<AttributeMatch> match, out int? coveredByUserId, out int? coveredByPartyId)
+        {
+            bool validUser = TryGetCoveredByUserIdFromMatch(match, out int coveredByUserIdTemp);
+            bool validParty = TryGetCoveredByPartyIdFromMatch(match, out int coveredByPartyIdTemp);
+            coveredByPartyId = validParty ? coveredByPartyIdTemp : null;
+            coveredByUserId = validUser ? coveredByUserIdTemp : null;
+
+            if (validUser)
+            {
+                return coveredByUserIdTemp.ToString();
+            }
+            else if (validParty)
+            {
+                return coveredByPartyIdTemp.ToString();
+            }
+            else
+            {
+                return null;
+            }
```

Comment:
Why? The return is always CoveredBy but this could be eighter a userid or a partyid depending on the delegation if none of this exist it returns null and this is also in the same ballpark the match did not define any coveredby and this ia then an error case.

Reply:
I initially didn't look too closely on how it is used, but fear that the method can suddenly be used for some other purpose. It must be made more obvious that the method is returning a file path/name section for a policy file and not a value that can be used to identify coveredBy.

The issue might be hypothetical, but what if the party id of one party is suddenly the same as the user id of someone else.

Reply:
The partyids and userids should never be the same value but in this case they would already be put in the same policy file the only difference would be that the file would be larger to parse at it contains two different covered by scopes. 

The reason for userids and partyids not the same is the start index for userid is 20 000 000 and for partyid it is 50 000 000 giving room for 30 000 000 userids before they hit the same scope and we do only have 5 408 918 userids used. One day this will be å potential overlap but for this case not a problem.

Will fix the documentation to say it is helper method for creating policypath out of userid/partyid